### DISCUSSION TITLE
Can I clone your website?

### DISCUSSION THREAD

**Discussion by coinlordd** [supabase/supabase#297] [Questions] [answered]
I'm working on a project and we're gonna use the Docusaurus lib to build our wiki page (for commercial use), and noticed your docs/website is included in https://v2.docusaurus.io/showcase/ and we love the design for our blueprint. Is it okay if we clone your website? I saw a license in the src dir but not in the website folder so i am curious if there are any gotcha's before using it? Thanks!

**Comment by kiwicopple** [comment, marked as answer]
Yes of course! Consider it also Apache-2.0, same as the license in the root.

We'd love to see what you build with it 👍
